GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/SecurityController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using static Utils.Server;
+
+public class SecurityController
```

Comment:
It handles the RPC from/to server so name it as RpcController or NetworkRequestController.

Reply:
I see, I like the NetworkRequestController name, will change it.